request.json

```diff
@@ -0,0 +1 @@
+{"url":"https://api.github.com/repos/puppetlabs/puppetlabs-satellite_pe_tools/actions/runs/5251671377","VMs":[{"cloud":null,"region":null,"zone":null,"images":["centos-stream-8"]}]}
```

Comment:
this file made it in by mistake

Reply:
yeah, its by mistake. Thanks for eagle eyes.